frontend/www/js/omegaup/time.ts

```diff
@@ -251,3 +251,28 @@ export function remoteTimeAdapter<T>(value: T): T {
   }
   return value;
 }
+
+export function formatContestDuration(
+  startDate: Date,
+  finishDate: Date,
+): string {
+  const startDateJs = dayjs(startDate);
+  const finishDateJs = dayjs(finishDate);
+
+  const durationInMiliSeconds: number = finishDateJs.diff(startDateJs);
+
+  const seconds: number = (durationInMiliSeconds / 1000) % 60;
+  const minutes: number = (durationInMiliSeconds / (1000 * 60)) % 60;
+  const hours: number = (durationInMiliSeconds / (1000 * 60 * 60)) % 24;
+
+  return (
+    (hours < 10 ? '0' : '') +
+    hours +
+    ':' +
+    (minutes < 10 ? '0' : '') +
+    minutes +
+    ':' +
+    (seconds < 10 ? '0' : '') +
+    seconds
+  );
```

Comment:
```suggestion
  return formatDelta(finishDate.getTime() - startDate.getTime());
```

ja, resulta que ni si quiera necesitamos la librería. originalmente había entendido que la intención era desplegarlo distinto, pero parece que no entendí bien lo que se buscaba en la reunión.

se le puede dar revert a la inclusión de `dayjs` y se puede en otro cambio reemplazar el uso de `moment` por `dayjs`? así ya no tendremos la tentación en el futuro.

Reply:
Sería de nuevo con el mismo comando `git restore --source=$commitViejo package.json yarn.lock` cierto?

Reply:
correcto!